Formula/vowpal-wabbit.rb

```diff
@@ -30,6 +31,7 @@ def install
                            "--with-boost=#{Formula["boost"].opt_prefix}"
     system "make"
     system "make", "install"
+    bin.install Dir["utl/*"] - ["utl/active_interactor.py", "utl/new_version", "utl/vw-validate.html"]
```

Comment:
Maybe just install them all and then use `rm` to delete them? A bit easier to follow, I think.